projects/ag-grid/documentation/code-examples/data-entry-grid/basic/data-entry-grid-docs-demo.component.ts

```diff
@@ -90,12 +90,14 @@ export class SkyDataEntryGridDemoComponent {
   public gridApi: GridApi | undefined;
   public gridOptions: GridOptions;
   public searchText: string = '';
+  public noRowsTemplate;
 
   constructor(
     private agGridService: SkyAgGridService,
     private modalService: SkyModalService,
     private changeDetection: ChangeDetectorRef
   ) {
+    this.noRowsTemplate = `No results found`;
```

Comment:
```suggestion
    this.noRowsTemplate = `No results found.`;
```
This is a complete sentence, so I'd add a period to the end. ... I'm having trouble serving locally to see the text in place, but I assume this is text to display in the grid when someone enters search criteria that doesn't have any matches. Is that correct?

Reply:
Sure thing, I'll add the period. 

Let's Huddle and I'll show you how to see it locally.

Yes, that is correct.

Reply:
fixed in commit [56545e6 - "add periods."](https://github.com/blackbaud/skyux-ag-grid/pull/193/commits/56545e6d112a5dc513982f5560e10cdc4a119b42)